packages/xdg_directories/test/xdg_directories_test.dart

```diff
@@ -76,7 +74,7 @@ XDG_VIDEOS_DIR="$HOME/Videos"
     expect(xdg.cacheHome.path, equals(testPath('.test_cache')));
     expect(xdg.configHome.path, equals(testPath('.test_config')));
     expect(xdg.dataHome.path, equals(testPath('.local/test_share')));
-    expect(xdg.runtimeDir.path, equals(testPath('.local/test_runtime')));
+    expect(xdg.runtimeDir!.path, equals(testPath('.local/test_runtime')));
```

Comment:
This `!` is not correct; the API can return a null. You need to validate both that it's not null, and that it matches if it's not.